Managers/RandomGenerator.gd

```diff
@@ -2,8 +2,17 @@ extends Node
 ## Initializes randomness for the game.
 
 
+const GAME_SEED = -1
```

Comment:
if this is `const` it means the `GAME_SEED` will always be at -1 when entering `_ready`, which means it will always be randomly set in `set_game_seed`. So we can't actually choose the seed.

Reply:
Yeah, we don’t have a UI or anything for this, so if you want to set the seed you can just modify the const

Reply:
maybe a little comment above to ask people to set the const back to -1 when they are done with testing.